src/components/Personalization/turbine/initRuleComponentModules.js

```diff
@@ -45,7 +46,6 @@ export default collect => {
     replaceHtml: createAction(collect, replaceHtml),
     prependHtml: createAction(collect, prependHtml),
     appendHtml: createAction(collect, appendHtml),
-    // TODO: NOT IMPLEMENTED
-    click: () => {}
+    click: settings => click(settings, store)
   };
 };
```

Comment:
I'm wondering if there is a better word than click here.  All of these are verbs or verb phrases, as is click, but for the other commands they are telling the client code what to do.  In the case of click, we are telling the client code what to collect when something is clicked.  What if we changed this to `collectClicks`?

Reply:
@jonsnyder it is already part of Konductor response, so to change it I would have to change Konductor, not a big deal, but I would like to leave it as is fir now